open_humans/settings.py

```diff
@@ -386,14 +350,22 @@ def __str__(self):
 
 AUTH_USER_MODEL = 'open_humans.User'
 
-ACCOUNT_LOGIN_REDIRECT_URL = LOGIN_REDIRECT_URL
-ACCOUNT_OPEN_SIGNUP = to_bool('ACCOUNT_OPEN_SIGNUP', 'true')
-ACCOUNT_PASSWORD_MIN_LEN = 8
-ACCOUNT_SIGNUP_REDIRECT_URL = 'home'
-ACCOUNT_HOOKSET = 'open_humans.hooksets.OpenHumansHookSet'
-ACCOUNT_EMAIL_CONFIRMATION_AUTHENTICATED_REDIRECT_URL = 'home'
-ACCOUNT_EMAIL_CONFIRMATION_ANONYMOUS_REDIRECT_URL = 'home'
-ACCOUNT_USE_AUTH_AUTHENTICATE = True
+ACCOUNT_AUTHENTICATED_LOGIN_REDIRECTS = True
+# Due to our user model, this basically gets ignored
```

Comment:
maybe more explanation like ...?
`# currently ignored due to custom User and ModelBackend (see above)`